exporter/clickhouseexporter/internal/metrics_model.go

```diff
@@ -18,16 +18,18 @@ import (
 	"go.uber.org/zap"
 )
 
-var supportedMetricTypes = map[string]struct{}{
-	createGaugeTableSQL:        {},
-	createSumTableSQL:          {},
-	createHistogramTableSQL:    {},
-	createExpHistogramTableSQL: {},
-	createSummaryTableSQL:      {},
+var supportedMetricTypes = MetricTableNames{
```

Comment:
The name of this type is a little confusing as it conflicts with the one from the config. It also obscures the idea that this is a mapping of supported metrics types, which suggests the caller may try to check whether a type is supported or not. I don't mind aliasing the type, but maybe we can rename it to differentiate it from the configuration type

Reply:
Another thought is, if we're going to change the value type of this mapping, it would be nice to have it map to a metric configuration type. 

I see the usage of this map below-- it maps directly to a table name. If we mapped it to a struct instead we could change the table name as well as other configuration options per table.

This is more of a future change though, nothing we need to change now